app/main.py

```diff
@@ -1,3 +1,57 @@
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: Union[int, float]) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Union["Distance", int, float]) -> "Distance":
```

Comment:
Improve your type annotations to not use `Union` and declare classes in `""` for return types. You could use `from future import __annotations__` import to achieve that.